### DISCUSSION TITLE
Elastic Queue Problems

### DISCUSSION THREAD

**Discussion by dunakov** [Jeroen-G/Explorer#124] [General]
Hi, When i use Laravel Scout option "queue" => 'true'  i have a error in my queue docker container No alive nodes found in your cluster, but  everything work without queue. 

I enter the docker-container and ping + Curl elasticsearch host, Elastic works and all ok. But i don't understand problem.....

**Comment by Jeroen-G** [comment]
When you have the "No alive nodes found in your cluster" error, what do the docker logs say?

**Comment by dunakov** [comment]
Docker logs it's ok. I had this problem before in  normal use, I fixed it  ElasticSearch option  "xpack.security.enabled=false". Now i am having the queue problem and i don't know why it's happened

**Comment by Jeroen-G** [comment]
I don't understand it either, but it seems not to be related to this package.

**Comment by idanieldrew** [comment]
This problem occurs when host address is wrong.[(here)](https://github.com/Jeroen-G/Explorer/blob/master/config/explorer.php#L12)
Anyway,if you think the address is correct and the data is not importent,you can delete index and import again.

**Reply by dunakov** [reply]
My Index worked before queue, but when i use any job my connection crush 

 My config is true
`'connection' => [
        'host' => env('ELASTICSEARCH_HOST'),
        'port' => '9200',
    ],`

**Comment by dunakov** [comment]
The problem resolved itself.......

**Reply by Jeroen-G** [reply]
Technology is magical 😂

**Reply by dunakov** [reply]
> Technology is magical 😂

:D Yes
